.gitignore

```diff
@@ -1,3 +1,5 @@
 vendor
+.idea
```

Comment:
@rajeshreeputra line# 2 & 3 isn't same? If we only add .idea then I guess it would work

Reply:
We should keep `.idea/` one

Reply:
Updated the same to keep .idea/